modules/reference/pages/systems/system-details.adoc

```diff
@@ -185,14 +185,14 @@ Contact Method::
 Available methods: Default (Pull), Push via SSH, and Push via SSH tunnel.
 +
 
-The so-called OSA status is also displayed for client systems registered with {productname} that have the OSA dispatcher (osad) configured.
+The so-called OSA status is also displayed for client systems registered with {productname} that have the OSA dispatcher (mgr-osad) configured.
```

Comment:
Maybe this should be `osad`, if we are talking about the service. Or keep it as `mgr-osad` if you are talking about the package.

Reply:
Depends. Do you configure a service or a package? ;)

I'll change it, only because I think 'osad' is more widely recognised.